Update toolbar docs (#5950)
PostHog/posthog.com: contents/docs/product-analytics/toolbar.mdx

In simple terms, the Toolbar is like 'Inspect Element' in a dev tool, but for user behavior.

> When enabled, the Toolbar appears as an overlay **on your website** (it is not visible to your users) and you will be able to seamlessly interact with the elements on your page to create Actions, get information about the most frequently triggered events, visualize statistics via a heatmap, and more.

Reviewer: ```suggestion
When enabled, the Toolbar appears as an overlay **on your website** (it is not visible to your users) and you can interact with the elements on your page to [create Actions](/tutorials/how-to-capture-events-the-easy-ways), get information about the frequency of events, visualize statistics via a heatmap, and more.
```

![PostHog toolbar](../../images/docs/product-analytics/toolbar.png)

> For an in-depth tutorial on how to visualize user behavior, check out our [Toolbar tutorial](/docs/tutorials/toolbar)

## How to activate the Toolbar

1. Go to 'Launch Toolbar' in the left-hand menu
2. Click 'Enable PostHog Toolbar'
3. Add the URLs you wish to enable the toolbar for. Only you will be able to see it (and not your users).
4. Click Save and then Launch.    
5. A new window will open with toolbar on your website. Click on the toolbar to interact with it.

> ![How to setup the PostHog toolbar](../../images/docs/product-analytics/toolbar-setup.png)

Reviewer: Sent you a video you could use here that might explain better.
